src/PostgREST/Parsers.hs

```diff
@@ -0,0 +1,175 @@
+module PostgREST.Parsers
+( parseGetRequest
+)
+where
+
+import           Control.Applicative hiding ((<$>))
+import           Data.Functor ((<$>))
+import           Data.Traversable (traverse)
```

Comment:
I added those to make ghc 7.8 happy.
https://circleci.com/gh/begriffs/postgrest/396
If it's changed to compile under 7.10 then those lines can be removed